lms/djangoapps/dashboard/sysadmin.py

```diff
@@ -622,16 +623,10 @@ def get(self, request, *args, **kwargs):
                 raise Http404
             cilset = CourseImportLog.objects.order_by('-created')
         else:
-            try:
-                course = get_course_by_id(course_id)
```

Comment:
this was useless, if course id is invalid i am just displaying message that `'No git import logs have been recorded for this course.`